.changes/unreleased/Bugfix-20230810-114939.yaml

```diff
@@ -0,0 +1,4 @@
+kind: Bugfix
+body: 'BREAKING CHANGE: to disable interactive UI pass the --noninteractive flag when
```

Comment:
@taimoor-at-opslevel - thinking about this more and reading it like this - i think we maybe have this backwards.  It feels like the default should be "non-interactive".  Thoughts?